domains/certificates/Query.API/API/MeasurementsSyncer/MeasurementsSyncService.cs

```diff
@@ -34,13 +37,20 @@ public MeasurementsSyncService(ILogger<MeasurementsSyncService> logger, ISliding
         _measurementSyncMetrics = measurementSyncMetrics;
         _measurementSyncPublisher = measurementSyncPublisher;
         _meteringPointsClient = meteringPointsClient;
+        _options = _measurementSyncOptions;
     }
 
     public async Task FetchAndPublishMeasurements(MeteringPointSyncInfo syncInfo, MeteringPointTimeSeriesSlidingWindow slidingWindow,
         CancellationToken stoppingToken)
     {
         var synchronizationPoint = UnixTimestamp.Now().RoundToLatestHour();
-        var fetchedMeasurements = await FetchMeasurements(slidingWindow, syncInfo.MeteringPointOwner, synchronizationPoint, stoppingToken);
+        var fetchToTimestamp = UnixTimestamp.Min(synchronizationPoint, synchronizationPoint.Add(TimeSpan.FromHours(-_options.Value.MinimumAgeThresholdHours)).RoundToLatestHour());
+        if (_options.Value.MinimumAgeThresholdHours > 0)
+        {
+            synchronizationPoint = UnixTimestamp.Now().Add(TimeSpan.FromHours(-_options.Value.MinimumAgeThresholdHours)).RoundToLatestHour();
```

Comment:
This looks like the same value used as second parameter in line 47, an idea could be to introduce a new variable for this value along synchronizationPoint.

Reply:
Already fixed. Can this be resolved?